project_event/models/calendar_event.py

```diff
@@ -0,0 +1,22 @@
+# © 2018 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class CalendarEvent(models.Model):
+    _inherit = 'calendar.event'
+
+    def default_category_id(self):
+        return self.env['task.category'].search(
```

Comment:
handle the case when there is no records in the table and maybe the reference would be more precise with a ref(rec_id